frontend/locales/en/strings.json

```diff
@@ -271,6 +267,23 @@
         "scriptWithArgs": "You must specify a script path if you specify args.",
         "customAmiSelect": "You must select an AMI ID if you enable Custom AMI."
       },
+      "container": {
+        "title": "Queues"
+      },
+      "slurmMemorySettings": {
+        "container": {
+          "title": "Slurm Memory Settings",
+          "info": "info"
+        },
+        "toggle": {
+          "label": "Slurm Memory Based Scheduling"
+        },
+        "info": {
+          "header": "Slurm Memory Based Scheduling only allows one instance per Queue",
+          "body": "If more than one queue is needed, disable Slurm Memory Based Scheduling option.",
```

Comment:
issue: the flag is disabled when selecting multiple instance types, not queues.

Reply:
Yes, my bad, copied directly from Figma design and forgot to amend.